smac_planner/src/node_se2.cpp

```diff
@@ -76,6 +81,9 @@ void MotionTable::initDubin(
   projections.emplace_back(hypotf(delta_x, delta_y), 0.0, 0.0);  // Forward
   projections.emplace_back(delta_x, delta_y, increments);  // Left
   projections.emplace_back(delta_x, -delta_y, -increments);  // Right
+
+  // Create the correct OMPL state space
+  state_space = std::make_shared<ompl::base::DubinsStateSpace>(min_turning_radius);
```

Comment:
Can we make these unique ptrs? Moderately less overhead.

Reply:
Done.